docs/generated/galasactl_streams.md

```diff
@@ -0,0 +1,29 @@
+## galasactl streams
+
+Manages test streams in an ecosystem
+
+### Synopsis
+
+Allows interaction with the streams servlet to return information about test streams.
```

Comment:
Instead of streams servlet, maybe something like "Parent command for managing test streams in a Galasa service"? Or something better?